youtube-music/build.gradle.kts

```diff
@@ -18,6 +18,7 @@ dependencies {
     implementation(libs.ktor.client.encoding)
     implementation(libs.ktor.client.serialization)
     implementation(libs.ktor.serialization.json)
+    implementation("com.squareup.okhttp3:logging-interceptor:4.10.0")
```

Comment:
do we really need this logging? Let's not add it because it was just for testing